Remotion/Web/Core/UI/Controls/DropDownMenuImplementation/Rendering/DropDownMenuRenderer.cs

```diff
@@ -415,20 +416,19 @@ private void AppendMenuItem (DropDownMenuRenderingContext renderingContext, Stri
                                    || menuItem.Command.Type == CommandType.WxeFunction;
           if (isPostBackCommand)
           {
-            // Clientside script creates an anchor with href="#" and onclick=function
-            // The function will be executed using eval(...) and must therefor not end with a return statement.
             string argument = menuItemIndex.ToString();
-            href = renderingContext.Control.Page!.ClientScript.GetPostBackClientHyperlink(renderingContext.Control, argument);
-            href = ScriptUtility.EscapeClientScript(href);
-            href = "'" + href + "'";
+            clickHandler = "function() { " + renderingContext.Control.Page!.ClientScript.GetPostBackEventReference(renderingContext.Control, argument) + "; return false; }";
+            href = "'" + ScriptUtility.EscapeClientScript(_fallbackNavigationUrlProvider.GetURL()) + "'";
```

Comment:
Don't know if we want to change it here in the new code but I think that using raw string literals could improve the readability here.

```c#
clickHandler = $$"""
      function() {
           {{renderingContext.Control.Page!.ClientScript.GetPostBackEventReference(renderingContext.Control, argument)}};
           return false;
      }
      """
```

Similar argument could be made at all the other string concatenations we added. 

A bit tricky to get the alignment right though :D

Reply:
Since the script is rendered as a parameter in the rest of the Javascript, I actually prefer the single-line version. Otherweise we get a ton of weird linebreaks in the rendered page. It's not something I explciitly went for (it just happened due to existing code styles) but it does have its benefits.